src/components/Navbar/Logo.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const NavbarLogo = () => {
+  return (
+    <div className="navbar__logo">
+      {/* TODO: Use logo instead*/}
```

Comment:
Do we show the logo in every page size or only on the mobile size?

Reply:
Per our last conversation, we need two logos in total, one for mobile and another for desktop. Thus, we will handle the logic when we have all the logos since we can simply do this
`<img src="small.jpg" srcset="medium.jpg 1000w, large.jpg 2000w" alt="yah">`